app/models/gift.rb

```diff
@@ -22,14 +22,16 @@ def has_name_or_pseud
     end
   end
 
-  validates_uniqueness_of :pseud_id, scope: [:work_id], message: ts("You can't give a gift to the same person twice.")
+  validates_uniqueness_of :pseud_id,
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.